test/integration/utils/builds.go

```diff
@@ -0,0 +1,45 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// This class is intended to host all CRUD calls for testing Build CRDs resources
+
+// CreateBuild generates a Build on the current test namespace
+func (t *TestBuild) CreateBuild(build *v1alpha1.Build) error {
+	bInterface := t.BuildClientSet.BuildV1alpha1().Builds(t.Namespace)
+
+	_, err := bInterface.Create(build)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
We could save a couple of lines here and in other Go source files. If there is nothing else after the create, we can just return the `err`. If it is `nil`, we just return `nil` as-is.
```suggestion
	return err
```